src/EffectiveCSharp.Analyzers/PreferDeclarationInitializersToAssignmentStatementsCodeFixProvider.cs

```diff
@@ -0,0 +1,257 @@
+namespace EffectiveCSharp.Analyzers;
+
+/// <summary>
+/// A <see cref="CodeFixProvider"/> that provides a code fix for the <see cref="PreferDeclarationInitializersToAssignmentStatementsAnalyzer"/>.
+/// </summary>
+/// <seealso cref="Microsoft.CodeAnalysis.CodeFixes.CodeFixProvider" />
+[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(PreferDeclarationInitializersToAssignmentStatementsCodeFixProvider))]
+[Shared]
+public class PreferDeclarationInitializersToAssignmentStatementsCodeFixProvider : CodeFixProvider
+{
+    private static readonly IEqualityComparer<IFieldSymbol> _fieldSymbolNameComparer = new FieldSymbolNameComparer();
```

Comment:
Why is this cached? It's fairly cheap to create

Reply:
I didn't see the need to recreate it multiple times and its not like the class was full of fields or properties that it created clutter. I'd rather just keep it as is

Reply:
Not needed anymore